charms/jupyter-ui/src/charm.py

```diff
@@ -169,6 +169,7 @@ def _get_env_vars(self):
             "UI": config["ui"],
             "USERID_HEADER": "kubeflow-userid",
             "USERID_PREFIX": "",
+            "METRICS": 1,
```

Comment:
I think instead of just enabling metrics like that, we should enable this in a separate `feature` PR alongside a dashboard and metrics. This change should be identical to what we did for kubeflow-volumes in https://github.com/canonical/kubeflow-volumes-operator/pull/176. In that case we also used a `dev` branch, but this is probably redundant since we 're not breaking something.